proposals/lip-0045.md

```diff
@@ -216,7 +216,7 @@ In this section, we specify the substores that are part of the Interoperability
 | `CHAIN_REGISTRATION_FEE` | uint64 | 1000000000 | Fee to pay for a sidechain registration command in Beddows. |
 | `EMPTY_HASH` | bytes | `sha256(b"")` | SHA-256 hash of empty bytes. |
 | `EMPTY_CCM` | CCM | A CCM object partially following the [`crossChainMessageSchema`][lip-0049#ccmschema] schema with all properties set to their default values (see exact definition [below](#empty-cross-chain-message)). | The empty ccm object. |
-| `LIVENESS_LIMIT` | uint32 | 30x24x3600 | The maximum time interval for the liveness condition. |
+| `LIVENESS_LIMIT` | uint32 | 28x24x3600 | The maximum time interval (in seconds) for the liveness condition. |
```

Comment:
```suggestion
| `LIVENESS_LIMIT` | uint32 | 28 * 24 * 3600 | The maximum time interval (in seconds) for the liveness condition. |
```

Reply:
I would strongly suggest to use `*` for multiplication with spacing as then it can be easily copied into a calculator or code instead of abusing the character `x` to achieve a certain visual output. Same below.

Reply:
Right, good catch. In LIP 45 for the liveness "x" was used and I just pasted, but I agree, it is much nicer in the way you suggest.